parsl/tests/configs/htex_local_radio.py

```diff
@@ -0,0 +1,74 @@
+"""
```

Comment:
this file is a an outdated copy of htex_local_alternate.py with no changes related to this PR, right?

Reply:
Oh yeah, sorry about that